utilities/rocAL/rocAL_unittests/rocAL_unittests.cpp

```diff
@@ -802,34 +805,34 @@ int test(int test_case, int reader_type, const char *path, const char *outName,
     printf("Remaining images %lu \n", rocalGetRemainingImages(handle));
     high_resolution_clock::time_point t1 = high_resolution_clock::now();
     int index = 0;
-
+    bool first_run = true;
     while (rocalGetRemainingImages(handle) >= inputBatchSize)
     {
         index++;
         if (rocalRun(handle) != 0)
             break;
-        int label_id[inputBatchSize];
         int numOfClasses = 0;
         int image_name_length[inputBatchSize];
         switch(pipeline_type)
         {
             case 1: //classification pipeline
             {
-                rocalGetImageLabels(handle, label_id);
+                RocalTensorList labels = rocalGetImageLabels(handle);
+                int * label_id = reinterpret_cast<int *>(labels->at(0)->buffer());  // The labels are present contiguously in memory
```

Comment:
Please remove the additional space after *

Reply:
Done